openmc/data/multipole.py

```diff
@@ -1,14 +1,24 @@
 from numbers import Real
 from math import exp, erf, pi, sqrt
+from copy import deepcopy
+import warnings
 
+import os
 import h5py
+import pickle
 import numpy as np
+from scipy.signal import find_peaks
+import matplotlib
+matplotlib.use("agg")
```

Comment:
This seems intrusive to me -- anyone who imports openmc.data would have their matplotlib backend potentially changed. Why does it need to be set manually here?

Reply:
Sorry, this was incorrectly copied from a notebook I guess. Just removed.